mosip_master/data_upgrade/1.2.0.1_to_1.3.0/data-uploader.py

```
@@ -0,0 +1,162 @@
+password=args.password
+
+def getCurrentDateTime():
+  dt_now = datetime.now(timezone.utc)
+  dt_now_str = dt_now.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3]
+  return dt_now_str+'Z'
+
+
+def get_seed_value():
+  conn = psycopg2.connect(database="mosip_master", user = args.dbusername, password = args.dbpassword, host = args.dbhost, port = args.dbport)
+  cursor = conn.cursor()
+  cursor.execute("select id from master."+args.table+" order by id desc limit 20")
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)